### DISCUSSION TITLE
Github page and viewport not working

### DISCUSSION THREAD

**Discussion by priyankadanda** [community/community#22919] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I am trying to host my portfolio on Github using Github pages.<br>
This is my repository: <a href="https://github.com/priyankadanda/priyankadanda.github.io" rel="nofollow noopener">https://github.com/priyankadanda/priyankadanda.github.io</a><br>
but when I visit <a href="https://priyankadanda.github.io/" rel="nofollow noopener">https://priyankadanda.github.io/</a>, It is saying “There isn’t a GitHub Pages site here.”<br>
The following is my index.html code for viewport,</p>
<pre><code>&lt;meta charset="utf-8"&gt;
&lt;meta name="viewport" content="width=device-width, initial-scale=1"&gt;
&lt;meta property="og:url" content="https://priyankadanda.github.io/" /&gt;
&lt;meta property="og:type" content="profile" /&gt;
&lt;meta property="og:title" content="Priyanka's Portfolio" /&gt;
&lt;meta property="og:description" content="Check out the stuff I've built Smiley Happy" /&gt;
&lt;meta property="og:image" content="https://i.imgur.com/iMHRTqT.png" /&gt;
&lt;meta name="theme-color" content="#007bff"&gt;
</code></pre>
<p>as you can see, the URL is established, but the problem persists. The viewport also shows a different image than the one described as property.</p>

**Comment by priyankadanda** [comment, marked as answer]
<p>It is fixed :slight_smile:</p>

**Comment by guygross** [comment]
How was it fixed?
